src/Microsoft.Fast.Components.FluentUI/Components/NumberField/FluentNumberField.razor.cs

```diff
@@ -59,13 +59,13 @@ public partial class FluentNumberField<TValue> : FluentInputBase<TValue>
     /// Gets or sets the maximum value
     /// </summary>
     [Parameter]
-    public string? Max { get; set; } = GetMaxValue();
+    public string? Max { get; set; } = InputHelpers<TValue>.GetMaxValue();
 
     /// <summary>
     /// Gets or sets the minimum value
     /// </summary>
     [Parameter]
-    public string? Min { get; set; } = GetMinValue();
+    public string? Min { get; set; } = InputHelpers<TValue>.GetMinValue();
```

Comment:
Since you are setting a generic component, it should be good to have typed properties: `TValue Step`, `TValue Max` and `TValue Min`. It's not logical to have numeric values and these properties as `string`.

Reply:
Agreed, but I ran into issues with the underlying web component when not using strings. Will revise in v3 branch

Reply:
Yes, but it's the Web Component that needs a "string" value. On our Blazor side, we need to type the property.